phable_cli/cli.py

```diff
@@ -186,9 +186,13 @@ def assign_task(ctx, task_ids: list[int], username: str | None):
     required=True,
     help="Name of destination column on the current project board",
 )
+@click.option(
+    "--milestone/--no-milestone",
+    default=True
```

Comment:
I think having `--no-milestone` be the default is actually a bit less surprising. WDYT?

Reply:
I was thinking about not breaking compatibility and that most users are probably going to want to use "--milestone". But fine with me!